src/core/SablierLockup.sol

```diff
@@ -408,36 +433,47 @@ contract SablierLockup is ISablierLockup, SablierLockupBase {
         });
     }
 
+    struct CreateLLParams {
```

Comment:
temporarily added this due to stack too deep error

will try to think of a better option, but couldn't find one yet (tried scoping `{}` blocks of code, tried a different function for the event, tried splitting the logic, and didn't work unfortunetely)

Reply:
https://github.com/sablier-labs/v2-core/pull/1075/commits/5764dd34aa56f544ff1e5caaa0be00bf9f9a610e